network-api/networkapi/templates/pages/base.html

```diff
@@ -110,8 +110,8 @@
         {% endif %}
 
         {% block donate_banner %}
-            {% if settings.donate_banner.SiteDonateBanner.active_donate_banner %}
-                {% include "fragments/donate_banner.html" with banner=settings.donate_banner.SiteDonateBanner.active_donate_banner.localized %}
+            {% if donate_banner %}
```

Comment:
Instead of displaying the donate banner set in the site’s settings, we are now rendering the banner provided to the context by the `get_donate_banner()` method.